lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -65,14 +65,31 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
     active_closures = Enum.flat_map(active, &elevator_closure/1)
     at_this_elevator? = fn %Closure{id: id} -> id == elevator_id end
 
+    {:ok, %Stop{id: stop_id}} = @facility.fetch_stop_for_facility(elevator_id)
+
+    relevant_closures =
+      active_closures |> Enum.filter(&relevant_closure?(&1, stop_id, active_closures))
+
     case Enum.find(active_closures, at_this_elevator?) do
       nil ->
         upcoming_closures =
           upcoming |> Enum.flat_map(&elevator_closure/1) |> Enum.filter(at_this_elevator?)
 
         [
-          elevator_closures(active_closures, upcoming_closures, app_params, now)
-          | header_footer_instances(config, now)
+          elevator_closures(
+            relevant_closures,
+            active_closures,
```

Comment:
I think we should only be passing in `relevant_closures` here, so all logic is consistently using the same set of closures.

Reply:
I had thought so too but this led to some test failures -- see my comment below